examples/annotate-apply-time-mutations-simple/README.md

```diff
@@ -0,0 +1,29 @@
+# annotate-apply-time-mutations: Simple Example
+
+### Overview
+
+This example shows how the [`annotate-apply-time-mutations`] function works.
+
+Running `annotate-apply-time-mutations` function on the example package will:
+
+1.  Set `config.k8s.io/apply-time-mutation` annotation on resources with `apply-time-mutation` inline comments.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/annotate-apply-time-mutations-simple
+```
+
+### Function invocation
+
+Invoke the function with the following command:
+
+```shell
+kpt fn eval annotate-apply-time-mutations-simple --image gcr.io/kpt-fn/annotate-apply-time-mutations:unstable
```

Comment:
This should land in gcr.io/kpt-fn-contrib repository.

Reply:
To achieve that, the function directory needs to be in `functions/contrib/go/` and the example directories needs to be in `examples/contrib/`. You should follow the same convention as for the curated catalog. e.g. example name should be `<fn-name>-<example-name>`.

Reply:
it looks like functions/contrib/go is a new directory, will that automatically get picked up by e2e tests and other CI/CD functions?

Reply:
all files moved to contrib, waiting for Makefile to validate e2e

Reply:
> it looks like functions/contrib/go is a new directory, will that automatically get picked up by e2e tests and other CI/CD functions?

I have pushed a Makefile for the new `functions/contrib/go` directory and update the root-level Makefile.
Things should be working now.